WORKSPACE

```diff
@@ -477,6 +477,3 @@ git_repository(
     remote = "https://github.com/nlohmann/json/",
     tag = "v3.11.3",
 )
-
```

Comment:
it is common practice to leave single blank line at the end of source file

Reply:
One line is left at the end, the diff shows wrong state.